Convex_hull_3/doc/Convex_hull_3/CGAL/Convex_hull_traits_3.h

```diff
@@ -42,7 +43,7 @@ typedef unspecified_type Plane_3;
 /*!
 
 */ 
-typedef Polyhedron_3<R> Polyhedron_3;
+typedef PolygonMesh Polygon_mesh;
```

Comment:
This is a breaking change.

It does even break the testsuite (seen in a local build of `integration`):
```
In file included from .../cgal/Point_set_processing_3/include/CGAL/vcm_estimate_normals.h:24:0,
                 from .../cgal/Point_set_processing_3/include/CGAL/vcm_estimate_edges.h:24,
                 from .../cgal/Point_set_processing_3/test/Point_set_processing_3/vcm_all_test.cpp:2:
.../cgal/Point_set_processing_3/include/CGAL/internal/Voronoi_covariance_3/voronoi_covariance_3.h: In instantiation of ‘F& CGAL::Voronoi_covariance_3::internal::tessellate_and_intersect(const DT&, typename DT::Vertex_handle, const Sphere&, F&) [with DT = CGAL::Delaunay_triangulation_3<CGAL::Epick, CGAL::Default, CGAL::Default, CGAL::Default>; Sphere = CGAL::Voronoi_covariance_3::Sphere_discretization<CGAL::Epick>; F = CGAL::Voronoi_covariance_3::internal::Covariance_accumulator_3<double>; typename DT::Vertex_handle = CGAL::internal::CC_iterator<CGAL::Compact_container<CGAL::Triangulation_vertex_base_3<CGAL::Epick, CGAL::Triangulation_ds_vertex_base_3<CGAL::Triangulation_data_structure_3<CGAL::Triangulation_vertex_base_3<CGAL::Epick, CGAL::Triangulation_ds_vertex_base_3<void> >, CGAL::Triangulation_cell_base_3<CGAL::Epick, CGAL::Triangulation_ds_cell_base_3<void> >, CGAL::Sequential_tag> > >, CGAL::Default, CGAL::Default, CGAL::Default>, false>]’:
.../cgal/Point_set_processing_3/include/CGAL/internal/Voronoi_covariance_3/voronoi_covariance_3.h:210:58:   required from ‘std::array<typename DT::Geom_traits::FT, 6ul> CGAL::Voronoi_covariance_3::voronoi_covariance_3(const DT&, typename DT::Vertex_handle, const Sphere&) [with DT = CGAL::Delaunay_triangulation_3<CGAL::Epick, CGAL::Default, CGAL::Default, CGAL::Default>; Sphere = CGAL::Voronoi_covariance_3::Sphere_discretization<CGAL::Epick>; typename DT::Geom_traits::FT = double; typename DT::Vertex_handle = CGAL::internal::CC_iterator<CGAL::Compact_container<CGAL::Triangulation_vertex_base_3<CGAL::Epick, CGAL::Triangulation_ds_vertex_base_3<CGAL::Triangulation_data_structure_3<CGAL::Triangulation_vertex_base_3<CGAL::Epick, CGAL::Triangulation_ds_vertex_base_3<void> >, CGAL::Triangulation_cell_base_3<CGAL::Epick, CGAL::Triangulation_ds_cell_base_3<void> >, CGAL::Sequential_tag> > >, CGAL::Default, CGAL::Default, CGAL::Default>, false>]’
.../cgal/Point_set_processing_3/include/CGAL/vcm_estimate_normals.h:92:53:   required from ‘void CGAL::internal::vcm_offset(ForwardIterator, ForwardIterator, PointPMap, std::vector<ValueT>&, double, std::size_t, const K&) [with ForwardIterator = __gnu_cxx::__normal_iterator<CGAL::Point_3<CGAL::Epick>*, std::vector<CGAL::Point_3<CGAL::Epick> > >; PointPMap = CGAL::Identity_property_map<CGAL::Point_3<CGAL::Epick> >; K = CGAL::Epick; Covariance = std::array<double, 6ul>; std::size_t = long unsigned int]’
.../cgal/Point_set_processing_3/include/CGAL/vcm_estimate_normals.h:256:26:   required from ‘void CGAL::compute_vcm(ForwardIterator, ForwardIterator, PointPMap, std::vector<std::array<typename Kernel::FT, 6ul> >&, double, double, const Kernel&) [with ForwardIterator = __gnu_cxx::__normal_iterator<CGAL::Point_3<CGAL::Epick>*, std::vector<CGAL::Point_3<CGAL::Epick> > >; PointPMap = CGAL::Identity_property_map<CGAL::Point_3<CGAL::Epick> >; Kernel = CGAL::Epick; typename Kernel::FT = double]’
.../cgal/Point_set_processing_3/test/Point_set_processing_3/vcm_all_test.cpp:47:78:   required from here
.../cgal/Point_set_processing_3/include/CGAL/internal/Voronoi_covariance_3/voronoi_covariance_3.h:141:59: error: no type named ‘Polyhedron_3’ in ‘class CGAL::Convex_hull_traits_3<CGAL::Epick, CGAL::Default, CGAL::Boolean_tag<false> >’
                     typedef typename Traits::Polyhedron_3 Polyhedron;
                                                           ^~~~~~~~~~
```